<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InjectWidget</class>
 <widget class="QWidget" name="InjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>323</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>323</width>
    <height>357</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QPlainTextEdit" name="messageHistoryPlainTextEdit">
     <property name="minimumSize">
      <size>
       <width>305</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>305</width>
       <height>120</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="injectButton">
     <property name="minimumSize">
      <size>
       <width>301</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>305</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Inject</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>305</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>305</width>
       <height>61</height>
      </size>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="enableDebugPrivilegeCheckBox">
        <property name="text">
         <string>Enable Debug Privilege</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>305</width>
       <height>93</height>
      </size>
     </property>
     <property name="title">
      <string>Targets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>37</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="selectedPayloadLineEdit">
        <property name="minimumSize">
         <size>
          <width>4</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>37</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>37</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DLL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="dllBrowseButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="LazyLoadedComboBox" name="processListComboBox">
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>444</width>
          <height>22</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LazyLoadedComboBox</class>
   <extends>QComboBox</extends>
   <header>LazyLoadedComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
